ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandIdAssigner.java

```diff
@@ -112,7 +110,7 @@ private static CommandId getDropTypeCommandId(final DropType dropType) {
   private static CommandId getTerminateCommandId(final TerminateQuery terminateQuery) {
     return new CommandId(
         CommandId.Type.TERMINATE,
-        terminateQuery.getQueryId().toString(),
+        terminateQuery.getQueryId().map(QueryId::toString).orElse("ALL"),
```

Comment:
nit: constant for `"ALL"`

Reply:
I'm generally in favour of pulling such things out into constants. But only when the constant is more descriptive than the literal, e.g. `static final int DEFAULT_TIMEOUT = 10;` makes the meaning of `10` more obvious.

In this case adding a constant for `"ALL"` called `ALL`... I actually think this makes the code _less_ readable, as you need to go check the constant to see what the actual string is.

Reply:
Plus this will go once I split a `terminate all` into N `terminate x` calls.